interview/infra/llm/openai_client.py

```
@@ -0,0 +1,100 @@
+"""
+        if not self.use_llm:
+            return """1. 자기소개\n2. 지원동기\n3. 장점과 단점\n4. 직무 관련 경험\n5. 향후 커리어 목표"""
+        else:
+            messages = [
+                SystemMessage(content="너는 면접 질문을 생성하는 면접관이야."),
+                HumanMessage(content=prompt)
+            ]
+            
+            try:
+                response = self.llm(messages)
+                return response.content
```

Ruff (0.15.5)

[warning] Consider moving this statement to an `else` block

(TRY300)